modules/persistent-storage-csi-group-snapshots-overview.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * storage/container_storage_interface/persistent-storage-csi-group-snapshots.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="persistent-storage-csi-group-snapshots-overview_{context}"]
+= Overview of CSI volume group snapshots
+
+A _snapshot_ represents the state of the storage volume in a cluster at a particular point in time. Volume snapshots can be used to provision a new volume. 
+
+A _volume group snapshot_ uses a label selector to group multiple persistent volume claims for snapshotting. A volume group snapshot represents copies from multiple volumes that are taken at the same point-in-time. This can be useful for applications that contain multiple volumes.
+
+{product-title} supports Container Storage Interface (CSI) volume group snapshots by default. However, the only storage type in {product-title} that supports volume group snapshots is OpenShift Data Foundation.
+
+Volume group snapshots provide three new API objects for managing snapshots:
+
+* `VolumeGroupSnapshot`
++
+This object requests creation of a volume group snapshot for multiple persistent volume claims. It contains information about the volume group snapshot operation, such as the timestamp when the volume group snapshot was taken, and whether it is ready to use. The creation and deletion of this object represents a desire to create or delete a cluster resource (a group snapshot).
```

Comment:
May be we could replace `desire` with a more relevant word. I am not sure of the context. Maybe it means `leads to`? Also parentheses need to be removed.